functional_tests/steps/bundles.py

```diff
@@ -121,3 +126,166 @@ def the_user_can_see_the_bundle_details_with_creator(context: Context) -> None:
+    context.page.get_by_role("link", name="Add bundle").click()
+    context.page.get_by_role("textbox", name="Name*").fill(search_bundle.name)
+    context.page.get_by_role("button", name="Create").click()
+
+
+@step('the user can search for unknown bundle with response "{response}"')
+def search_for_new_bundle(context: Context, response: str) -> None:
+    unknown_bundle = "Unknown bundle"
+    if unknown_bundle not in get_bundles_names(context.bundles):
+        context.page.get_by_role("textbox", name="Search term").click()
+        context.page.get_by_role("textbox", name="Search term").fill(unknown_bundle)
+        expect(context.page.get_by_text(response)).to_be_visible()
+
+
+@step("the user can search for a known bundle")
+def search_for_existing_bundle(context: Context) -> None:
+    context.page.get_by_role("link", name="Bundles", exact=True).click()
+    context.page.get_by_role("textbox", name="Search term").click()
+    context.page.get_by_role("textbox", name="Search term").fill(context.bundles[0].name)
+    context.page.get_by_role("textbox", name="Search term").click()
+    expect(context.page.get_by_role("link", name=context.search_bundle)).to_be_visible()
+    expect(context.page.get_by_text("No bundles match your query.")).not_to_be_visible()
+    context.page.get_by_role("link", name=context.search_bundle).click()
+
+
+@step("the user can edit the known bundle")
+def can_edit_bundle(context: Context) -> None:
+    # goto bundles index
+    context.page.goto(context.base_url + reverse("bundle:index"))
+
+    # search for and goto Bundle edit
+    context.page.get_by_role("textbox", name="Search term").click()
+    context.page.get_by_role("textbox", name="Search term").fill(context.bundles[1].name)
+    context.page.get_by_role("link", name=context.bundles[0].name).click()
+
+    # add Release Calendar
+    expect(context.page.get_by_role("button", name="Choose Release Calendar page")).to_be_visible()
+    context.page.get_by_role("button", name="Choose Release Calendar page").click()
+    context.page.get_by_role("textbox", name="Search term").click()
+    context.page.get_by_role("textbox", name="Search term").fill(context.search_release_calendar.title())
+    expect(context.page.get_by_role("link", name=context.release_calendar_pages[1].title())).to_be_visible()
+    context.page.get_by_role("row", name=context.search_release_calendar.title).get_by_role("link").click()
+    context.page.get_by_role("link", name=context.search_release_calendar.title).click()
+
+    # modify status
+    context.page.locator("#id_status").select_option("IN_REVIEW")
+
+    # add preview team
+    context.page.get_by_role("button", name="Add preview team").click()
+    context.page.get_by_role("textbox", name="Search term").click()
+    context.page.get_by_role("textbox", name="Search term").fill(context.teams[1])
+    context.page.get_by_text(context.teams[1]).click()
+    context.page.get_by_role("button", name="Confirm selection").click()
```

Comment:
I think we should follow this with an assertion to confirm that we have added the preview team successfully.